shared/src/iosMain/kotlin/io/customer/shared/util/PlatformUtil.kt

```diff
@@ -0,0 +1,9 @@
+package io.customer.shared.util
+
+import platform.Foundation.NSUUID
+
+internal actual fun getDatabaseUtil(): DatabaseUtil = IOSDatabaseUtil()
+
+internal class IOSDatabaseUtil : DatabaseUtil {
```

Comment:
```suggestion
internal class iOSDatabaseUtil : DatabaseUtil {
```

Sorry, as someone who types `iOS` a lot every day this bothered me 😆

Reply:
I'm with you on this, it feels like `iOS` and `IOS` are two different things every time 😆 But since we both have same opinion now, I think we can agree to ignore class name with capitals conventions for the project whenever it starts with `iOS` 😄